beheer.md

```diff
@@ -39,48 +39,54 @@ Beheer helpt API developers/designers door:
   * ...
-* De OAS3 en de RI sluiten naadloos op elkaar aan.
-  * Dit kan door het steekproefsgewijs testen van Openapi.yaml vs test omgeving, vergelijk de resource
-* De testomgeving met een implementatie van de RI moet draaien. Dit betekent ook dat:
-  * De testscripts voor de RI waarmee we al het gewenst gedrag zoals beschreven in standaard testen ook beschikbaar zijn gesteld. Beheer zal dit bij doorontwikkeling weer nodig hebben en tevens zullen zij deze beschikbaar stellen aan de leveranciers/gemeenten. 
-* De docker container met de RI moet beschikbaar zijn op DockerHub in de VNG namespace.
-* Documentatie omvat minimaal:
+* De docker container met de RI moet beschikbaar zijn op DockerHub in de VNG namespace;
+* De documentatie omvat minimaal:
   * Een UML model van het informatiemodel (de objecttypen en onderlinge relaties) zoals deze in de API gebruikt wordt.
   * Een overzicht van welke gegevens worden ontsloten in de vorm van een tabel van de resource + bijzonderheden. 
-  * Installatie instructies van de referentie implementatie (bijv. INSTALL.rst in de repository).
-  * Een beschrijving van de business logica die in de standaard van toepassing is.
-  * Een overzicht van wijzigingen (bijv. CHANGELOG.rst in de repository) met daarin per versie wat is toegevoegd, gewijzigd en hoe te migreren van de voorgaande naar deze versie.
-  * De standaard bevat een overzicht waarin wordt aangegeven welke versies van de betreffende API standaard met welke versies van andere API standaarden compatible zijn.
- * De standaard bevat een overzicht van alle partijen en personen die bij de ontwikkeling van de standaard betrokken zijn (geweest).
+  * Installatie instructies van de RI (bijv. INSTALL.rst in de repository).
+* De testomgeving met een implementatie van de RI moet probleemloos draaien. Dit betekent ook dat:
+  * De testscripts voor de RI waarmee we al het gewenst gedrag zoals beschreven in standaard testen ook beschikbaar zijn gesteld. Beheer zal dit bij doorontwikkeling weer nodig hebben en tevens zullen zij deze beschikbaar stellen aan de leveranciers/gemeenten. 
 
 **Informatievoorziening**
-* De Github repositories behorende bij de standaard moet publiekelijk beschikbaar zijn;
-* De OAS moet zonder restricties op te vragen zijn;
-* Er is een mechanisme om vragen te kunnen stellen aan beheer, (bijvoorbeeld via Slack);
-* De website http://ref.tst.vng.cloud/ bevat een pagina over de standaard;
-* Gemma Online http://www.gemmaonline.nl/ bevat een pagina over de standaard;
-* In de toekomst bevat de website http://developer.overheid.nl/ een pagina over de standaard.
+
+Daarnaast worden er ook eisen gesteld aan de informatievoorziening over de API standaard. Zo moet:
+* de Github repositories behorende bij de standaard publiekelijk beschikbaar zijn;
+* de OAS zonder restricties op te vragen zijn;
+* er is een mechanisme zijn om vragen te kunnen stellen aan beheer, (bijvoorbeeld via Slack);
```

Comment:
er is een mechanisme zijn:
 er is een mechanisme 
of
 er moet een mechanisme zijn

Reply:
Is aangepast.